CONTRIBUTING.md

```diff
@@ -78,6 +78,14 @@ Back to common checking process:
 - Deploy Kepler to a local kind cluster with image stored at local image registry.
 - Use kubectl to check that the change was applied as intended.
   
+## Benchmark
```

Comment:
added usage base on my personal blog and https://blog.csdn.net/oe1019/article/details/112308242 relate reference.
I suppose we are able to generate prof and flame graph by golang benchmark.
cc: @rootfs , @marceloamaral